pkg/kubelet/nodestatus/setters_test.go

```diff
@@ -293,6 +293,94 @@ func TestNodeAddress(t *testing.T) {
 			hostnameOverride: true,
 			shouldError:      false,
 		},
+		{
+			name:   "Dual-stack cloud, IPv4 first, no nodeIP",
```

Comment:
gofmt is complaining for the spaces between `name:` and the description 🤷‍♂ on these test cases